File: php/DAVI/exercicio3.php
<!--3. Faça um programa que receba o peso de uma pessoa, calcule e exiba: o
peso dessa pessoa em gramas; se essa pessoa engordar 5% qual será seu
novo peso em gramas. Colocar tudo dentro de uma tabela conforme
exemplo acima.--> 
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
     <table border="1" align="center" width="50%">
        <tr><!--PRIMEIRA LINHA-->
            <td align="center"> <h1>EXERCÍCIO 3</h1> </td>
        </tr>
        <tr><!--SEGUNDA LINHA-->
            <td align="center"> <h2> AULA DS - 03/03/2026 </h2> </td>
        </tr>
        <tr><
            <td> <h2>
                <?php

                    $p=50;
                    $pg=$p*1000;
                    $npg=$pg*1.05;
                    print("Peso:$p kg<br>");
                    print("Peso em gramas:$pg g<br>");
                    print("Novo Peso em Gramas com 5%:$npg g<br>");
                ?>

                </h2> 
            </td>
        </tr>
        <tr>
            <td align="center"> <h2> Desenvolvido por DAVI</h2>
        </tr>
    


</body>
</html>

File: PYTHON/davi/exercicio3.py
"""3. Sabe-se que o quilowatt de energia custa um quinto do salário mínimo.
Faça um programa que receba o valor do salário mínimo e a quantidade de
quilowatts gasta por uma residência. Calcule e exiba:
 O valor, em reais, de cada quilowatt
 O valor, em reais, a ser pago por essa residência
 O novo valor a ser pago por essa residência, a partir de um desconto de
15%"""

print("=" * 50)
print("Exercicio 3 - LP - 04/03/2026".center(50))
print("=" * 50)




sm = int(input("Digigte o salário mínimo:"))
qw = int(input("Digite a quantiodade de quilowatts:"))

vq=sm*0.2
vr=vq*qw
nv=vr-(vr*0.15)

print("Valor quilowatt:",vq)
print("Valor a pagar:",vr)
print("Valor a pagar com desconto:",nv)

print("=" * 50)
print("Desenvolvido por Davi".center(50))
print("=" * 50)
